src/java.base/share/classes/java/text/DecimalFormat.java

```diff
@@ -417,6 +417,10 @@
  * <li>Exponential patterns may not contain grouping separators.
  * </ul>
  *
+ * @implNote The default implementation follows the LDML specification for
+ * {@code parseLenient} elements to interpret minus sign patterns when lenient
+ * parsing is enabled.
```

Comment:
IMO, the following is more clear.

`when lenient parsing is enabled` -> `when {@link #isStrict()} returns false`
`interpret minus sign patterns` -> `enable loose matching of minus sign patterns`

Also, I'm unsure on mentioning `{@code parseLenient} elements` because I'm not sure that users of DecimalFormat will be aware of such LMDL elements. This also seems to be the first time a direct mention of an LDML element is made. I'm not sure of a better alternative ATM.

Reply:
Thanks, modified as you suggested. For the `parseLenient`, the problem is in the LDML document side, as there is no description for it, and their loose matching description is kind of incorrect (use [:DASH:], which is not the case here. So I just dropped it in this iteration.

Reply:
I noticed that as well, so I could not think of a better alternative. I think dropping the element name is the right choice, less is more in this case.